public/app/features/dashboard-scene/saving/provisioned/SaveProvisionedDashboardForm.tsx

```diff
@@ -238,7 +249,7 @@ export function SaveProvisionedDashboardForm({
             'File path inside the repository (.json or .yaml)'
           )}
         >
-          <Input id="dashboard-path" {...register('path')} />
+          <Input id="dashboard-path" {...register('path')} readOnly={!isNew} />
```

Comment:
we do not yet have a way to rename files -- so lets make the path read only for now.

or we could hide it?  but I think better to be more explicit in the editor for now

Reply:
Agree, let's keep it as read-only.